packages/template-mrt-reference-app/app/ssr.test.js

```diff
@@ -32,7 +32,8 @@ describe('server', () => {
     test.each([
         ['/', 200, 'application/json; charset=utf-8'],
         ['/tls', 200, 'application/json; charset=utf-8'],
-        ['/exception', 500, 'text/html; charset=utf-8']
+        ['/exception', 500, 'text/html; charset=utf-8'],
+        ['/env-vars', 200, 'application/json; charset=utf-8']
```

Comment:
@josebarrios-sfdc – now that we've removed the `env-vars` specific endpoint, do we still need this additional test?

Reply:
My bad, I forgot to remove that test when removing the endpoint. All done now.